### ISSUE TITLE
Import auto-fix missing

### ISSUE THREAD

**Issue by surdu** [microsoft/vscode-eslint#1092]
I have the same issue as described in #914 . 

I've created a `jsconfig.json` containing:

```
{
  "compilerOptions": {
    "module": "commonjs",
    "target": "es6",
    "jsx": "preserve"
  },
  "exclude": [
    "node_modules",
    "**/node_modules/*"
  ]
}
```
but still, when I try to auto-fix an no-unused-var lint issue, there is no suggestion to import the variable.

@dbaeumer To answer your question from #914: no , the problem is not fixed if I run the linter with --fix. But the same auto-fix works for Typescript.

**Comment by dbaeumer** [comment]
@surdu I am pretty sure the auto fix is coming then from the TS support of VS Code not from ESLint. So this sounds more like an issue if the TS language server.

How does the error look like that gets auto fixed?

**Comment by surdu** [comment]
This is how the import error that gets fixed looks in Typescript:

<img width="902" alt="Screen Shot 2020-10-20 at 11 10 50 PM" src="https://user-images.githubusercontent.com/11520795/96638972-a6ebf300-1329-11eb-856d-99886ed1af29.png">

**Comment by dbaeumer** [comment]
Exactly how I thought it is. In the TS case the error is fixed by the TS/JS server. In the second case the error is reported by ESLin, but ESLint has no fix for it. I would suggest you open an issue against VS Code itself asking why the TS/JS server is not providing a corresponding fix for the JS file. 

I will close the issue since there is nothing the ESLint extension can do to fix this.
